docs/data/charts/bars/bars.md

```diff
@@ -85,6 +85,21 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 
 {{"demo": "GridDemo.js"}}
 
+### Color scale
+
+Like other charts, you can modify the [series color](/x/react-charts/styling/#colors) by using series color, or some color palette.
```

Comment:
```suggestion
Like for other charts, you can modify the [series color](/x/react-charts/styling/#colors) by using series color, or some color palette.
```

```suggestion
Like on other charts, you can modify the [series color](/x/react-charts/styling/#colors) by using series color, or some color palette.
```

With "Like other charts", I expect the sentence to be "Like other charts, the BarChart allows you...", but it's heavy